### DISCUSSION TITLE
hover over calendar

### DISCUSSION THREAD

**Discussion by wzprichardwzp** [sandboxnu/office-hours#693] [Ideas]
It would be nice if on the calendar schedule of office hour, when you click on a time block or hover over it, it would expend and show all the TAs. Currently if there's like more than 3 or 4, you can't really see the names of all of them.

**Comment by willstenzel** [comment]
That's a good idea, but given the library we use to render the calendar it's unlikely we will be able to implement this. We currently use the library `react-big-calendar` and I don't believe it has that level of customizability. However, if we change the calendar library in the future we will keep this in mind.
